pkg/controllers/status/workstatus_controller.go

```diff
@@ -275,10 +277,10 @@ func (c *WorkStatusController) recreateResourceIfNeeded(work *workv1alpha1.Work,
 
 // reflectStatus grabs cluster object's running status then updates to its owner object(Work).
 func (c *WorkStatusController) reflectStatus(work *workv1alpha1.Work, clusterObj *unstructured.Unstructured) error {
-	statusMap, _, err := unstructured.NestedMap(clusterObj.Object, "status")
+	statusRaw, err := c.ResourceInterpreter.ReflectStatus(clusterObj)
```

Comment:
If `statusRaw` == nil and the status in `Work` is nil too, then we can save one update operation.

There is a corner case that, the status in `Work` is not nil, but this time we get `nil` from the interpreter, then, we have to update the `Work`.

Reply:
How about adding an equal judgment when updating.